src/component/Board.jsx

```diff
@@ -62,22 +66,35 @@ function Board() {
     setMovesCounter(0);
   };
 
+  const scoreCalculation = () => {
+    baseScore = cardLevel / 2 - 1;
+    score = Math.round((baseScore / movesCounter) * 100);
+  };
+
   // function to check game is ended or not
   const gameEnd = () => {
+    scoreCalculation();
     if (pairCounter >= cardLevel / 2) {
-      setHeading("Congratulation you Won!");
-      setGameOver(true);
+      setHeading(`Hurrah! 🎉 You won with ${score}% accuracy`);
+      setTimeout(() => {
+        setGameOver(true);
+      }, 1500);
     }
   };
 
   // Starting game according to level or custome input
   const levelSetUp = (level) => {
     setCardLevel(level);
-    card = [];
+    // card = [];
+    card.length = 0;
```

Comment:
both are doing same thing, Why did you change `card = []` to ` card.length = 0`?